workflows/pipe-common/scripts/mount_storage.py

```diff
@@ -201,23 +203,27 @@ def get_path(self):
         return self.storage.path.replace(self.scheme(), '', 1)
 
     def _get_credentials(self, storage):
-        region_id = storage.region_id
-        account_id = os.getenv('CP_ACCOUNT_ID_{}'.format(region_id))
-        account_key = os.getenv('CP_ACCOUNT_KEY_{}'.format(region_id))
-        account_region = os.getenv('CP_ACCOUNT_REGION_{}'.format(region_id))
-        if not account_id or not account_key or not account_region:
-            raise RuntimeError('Account information wasn\'t found in the environment for account with id={}.'
-                               .format(region_id))
-        return account_id, account_key, account_region
+        return self._get_credentials_by_region_id(storage.region_id)
```

Comment:
since this logic is different for GCP, maybe it is better to override method in corresponding `GCPMounter`?

Reply:
yes, totally